Rocket.Chat/Managers/AppManager.swift

```diff
@@ -35,5 +35,4 @@ struct AppManager {
     static var supportsMultiServer: Bool {
         return applicationServerURL == nil
     }
-
```

Comment:
🤔

Reply:
A space disappeared when I switched from using a custom Info.plist to server values. I put it back 😛